saver.go

```diff
@@ -131,10 +141,14 @@ func (s *saver) QueryForInsert(modelPtr interface{}) (*SaveQuery, error) {
 		return nil, fmt.Errorf("obj doesn't have exql tags in any fields")
 	}
 
-	if !hasPrimary {
+	if len(primaryKeyFields) == 0 {
 		return nil, fmt.Errorf("table has no primary key")
 	}
 
+	if !hasAutoIncrementField {
```

Comment:
This case should be acceptable. E.x. there can be a model that has only manual primary key.

Reply:
Fixed in cc63a81